crates/sui-graphql-rpc/src/config.rs

```diff
@@ -10,9 +10,9 @@ use sui_json_rpc::name_service::NameServiceConfig;
 use crate::functional_group::FunctionalGroup;
 
 // TODO: calculate proper cost limits
-const MAX_QUERY_DEPTH: u32 = 20;
-const MAX_QUERY_NODES: u32 = 200;
-const MAX_QUERY_PAYLOAD_SIZE: u32 = 5_000;
+const MAX_QUERY_DEPTH: u32 = 15;
+const MAX_QUERY_NODES: u32 = 50;
```

Comment:
I think this limit in particular is going to cause issues for people by being way too small/conservative.  The test for PTBs that enumerates all the different kinds of PTBs needed this limit increased to 500, because it has a lot of input nodes, to describe all the possible different fields in all the different varieties of command.  This is a reasonable transaction to run (and one that the Explorer needs to perform).